File: Python/POO2/terminal/Terminal.py
class Terminal():

  def __init__(self, telefono):
    self.__telefono = telefono
    self.__duracion = 0

  def setTelefono(self, telefono):
    self.__telefono = telefono

  def getTelefono(self):
    return self.__telefono

  def setDuracion(self, duracion):
    self.__duracion = duracion

  def getDuracion(self):
    return self.__duracion

  def llamar (self, t, s):
    self.__duracion += s
    t.__duracion += s

  def __str__(self):
    return "Nº " + str(self.getTelefono()) + " - " + str(self.getDuracion()) + "s de conversación"

File: Java/POO2/terminal/Terminal.java
package terminal;

/**
 * Implementa la clase Terminal. Un terminal tiene asociado un número. Los
 * terminales se pueden llamar unos a otros y el tiempo de conversación corre
 * para ambos. A continuación se proporciona el contenido del main y el
 * resultado que debe aparecer por pantalla.
 * 
 * @author Juan Manuel Gonzalez Profumo
 * @version 1.0
 */
public class Terminal {
	private String telefono;
	private int duracion;

	public Terminal(String telefono) {
		this.telefono = telefono;
	}

	public String getTelefono() {
		return telefono;
	}

	public void setTelefono(String telefono) {
		this.telefono = telefono;
	}

	public int getDuracion() {
		return duracion;
	}

	public void setDuracion(int duracion) {
		this.duracion = duracion;
	}
	
	void llamar(Terminal t, int s) {
		this.duracion += s;
		t.duracion += s;
	}

	@Override
	public String toString() {
		return "Nº " + getTelefono() + " - " + getDuracion() + "s de conversación";
	}
	
}
